src/MessagePack/SequenceReader.cs

```diff
@@ -71,25 +71,6 @@ public SequenceReader(ReadOnlySequence<T> sequence)
             }
         }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="SequenceReader{T}"/> struct
-        /// over the given <see cref="ReadOnlyMemory{T}"/>.
-        /// </summary>
-        public SequenceReader(ReadOnlyMemory<T> memory)
```

Comment:
Removed ctor to make it inline with corefx

Reply:
Per my comment above, this wasn't done for fun. It was done because it made a significant perf improvement which was hard-earned. Not giving it up. Nor should we need to. We can keep using this type even on netcoreapp3.0.